journal-letters/modernized/16-on-stage-entertainment.adoc

```diff
@@ -1,12 +1,13 @@
 [short="Remarks on Attending Stage Entertainments"]
 == Remarks on the Prevailing Custom of Attending Stage Entertainments; also on the Present Taste for Reading Romances and Novels; and on some other Customs
 
-Submitted, with a heart overflowing with good-will,
-to the notice and consideration of the Professors of the Christian name,
-in the different religious Societies.
+[.blurb]
```

Comment:
Ok, so this whole intro section was pretty difficult.  I definitely think that this "Submitted..." chunk qualifies as a "Heading Continuation Blurb" (which is in the styleguide, but rarely used -- and often I mark those up before they get to you.

Anyway, I pulled this down and tried a few variations and I think I'll have you while your making the other changes, just change this whole section to what I paste below:

```ASCIIDOC
== Remarks on the Prevailing Custom of Attending Stage Entertainments

[.heading-continuation-blurb]
Also on the Present Taste for Reading Romances and Novels; and on some other Customs

[.heading-continuation-blurb]
Submitted, with a heart overflowing with good-will,
to the notice and consideration of the Professors of the Christian name,
in the different religious Societies.

[.centered]
_Vitarc peccatum, est vitare occasioncs peccati._

[.section-author-context]
Reprinted from the Third Edition.

=== Remarks, etc.
```